### ISSUE TITLE
[Project] Wanted! Japan needs foreign labour force

### ISSUE THREAD

**Issue by kidaemon** [jsoma/data-studio-projects#244]
Please complete all of the following sections, or the ghost of Joseph Pulitzer will spookily dance around your issue! A completed version of this template can be found at https://github.com/kidaemon/data-studio/tree/master/code/04-Project

# Pitch

## Summary

Due to the aging population and shortage of young labour force, Japanese government is now considering more open immigration policy. How urgent is the situation?

![](https://github.com/kidaemon/data-studio/blob/master/code/04-Project/Webp.net-gifmaker.gif)

## Details

**Possible headline(s):** 

**Data set(s):** 
Statistics Bureau, Ministry of Internal Affairs and Communications
http://www.stat.go.jp/english/data/roudou/lngindex.html

**Code repository:** 
https://github.com/kidaemon/data-studio/tree/master/code/04-Project
	
**Possible problems/fears/questions:** 

## Work so far

Japanese labour force is aging rapidly.

Graph1: Average age of labour force
![image](https://user-images.githubusercontent.com/39416343/43482859-e9906d58-94d7-11e8-82ee-6adaaa4601e8.png)

Graph2: Younger generation is shrinking
![image](https://user-images.githubusercontent.com/39416343/43482835-d0cb3410-94d7-11e8-9b79-65bf1a7c84a7.png)

https://github.com/kidaemon/data-studio/blob/master/code/04-Project/Webp.net-gifmaker.gif

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far

**Comment by sarahslo** [comment]
nice to see the age tree. i'd love to see it for each decade. i don't think you can say that the younger age group is shrinking because the chart doesn't show change.

what is the difference between the blue and orange line? and how is that possible that the average age is over 46?!

**Comment by vpenney** [comment]
I agree--a tree for each decade would be great, as well as some clarification between the two colors. Do they represent men vs women? 

Since you mention the immigration policy, maybe throw in some statistics about that as well. How many people are immigrating each year, and from where? Has that changed over time? 

Looking forward to seeing what you find!

**Comment by pasiegrist** [comment]
Hi there, the bot sent me here, and I am not sure if you are still working on this. Anyhow: I found the link you posted to the gif, and I like it a lot, as it shows the change pretty nice. But echoing the critique before , you should label it more clearly and I think slowing down speed of the GIF would help.
